app/metrics/jupyterkernelbusyness.go

```diff
@@ -0,0 +1,128 @@
+package metrics
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type JupyterKernelBusynessMetricsHandler struct {
+	logger         *logrus.Logger
```

Comment:
Please consider using nuclio.Logger, of which logrus is just an implementatioin

Reply:
Talked offline, logrus is not an implementation, I've created https://github.com/nuclio/loggerus (based on provazio code in provazio) that will be used instead logrus